echopype/utils/log.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import sys
+from typing import List, Optional
+
+LOG_FORMAT = "{asctime}:{name}:{levelname}: {message}"
+LOG_FORMATTER = logging.Formatter(LOG_FORMAT, style="{")
+STDOUT_NAME = "stdout_stream_handler"
+STDERR_NAME = "stderr_stream_handler"
+LOGFILE_HANDLE_NAME = "logfile_file_handler"
+
+
+class _ExcludeWarningsFilter(logging.Filter):
+    def filter(self, record):  # noqa
+        """Only lets through log messages with log level below ERROR ."""
+        return record.levelno < logging.WARNING
+
+
+def verbose(logfile: Optional[str] = None, override: Optional[bool] = None) -> None:
```

Comment:
As written, `override` has a default `None` value that will raise an error (line 33). Wouldn't it be more helpful to set the default to a boolean value? `False` seems the most logical to me: the user is calling a function whose name "verbose" suggests that it should make things verbose unless otherwise stated.

Reply:
It won't raise an error because I catch it by checking if `override is not None` then check if it's a boolean. On line 37 I have a check if `verbose` is `None` then by default set to `True`. The reason I do this logic is because I want the user to specifically call `override` if they'd like to override the actual verbose value. Otherwise `verbose` means that it will be verbose by default.